site/src/pages/components/selectlist.mdx

```diff
@@ -70,23 +70,23 @@ selectlist::button {
 
 ### Rich content in `<option>`s
 
-This example adds "rich content" inside option elements in the form of country flags. This is a common pattern on the web which isn't possible in the existing `<select>` element because `<select>`'s `<option>`s only render text content.
+This example adds "rich content" inside option elements in the form of images next to the options. This is a common pattern on the web which isn't possible in the existing `<select>` element because `<select>`'s `<option>`s only render text content.
 
 <div className="code-image-container">
 <div>
 ```html
 <selectlist>
   <option>
-    <img src="usa.jpg">
-    USA
+    <img src="swallow.jpg">
```

Comment:
Do these new images exist in the repo? I don't see them but maybe I missed it.

Reply:
don't forget `alt=""` or `alt`

Reply:
I switched back to country flags and added alt text.
The images don't exist in the repo, but they don't need to because they aren't actually downloaded or rendered anywhere on the website - the only exist in the demo which I made locally just to create the adjacent screenshot.